<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu::CTableMoving</class>
 <widget class="QGroupBox" name="MainMenu::CTableMoving">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>125</width>
    <height>306</height>
   </rect>
  </property>
  <property name="title">
   <string>Move</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="beginButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../Resource/MainMenu.qrc">
       <normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Up_End.png</normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Up_End.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>62</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="upButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../Resource/MainMenu.qrc">
       <normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Up.png</normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>62</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../Resource/MainMenu.qrc">
       <normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Down.png</normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>62</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="endButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="../Resource/MainMenu.qrc">
       <normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Down_End.png</normaloff>:/HimalayaImages/Icons/Move/Icon_Move_Down_End.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>62</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource/MainMenu.qrc"/>
 </resources>
 <connections/>
</ui>
